tests/test_utils/test_nan_warnings.py

```diff
@@ -0,0 +1,77 @@
+"""
+Test the RadViz feature analysis visualizers
+"""
+
+import unittest
+import numpy as np
+
+from yellowbrick.utils.nan_warnings import warn_if_nans_exist, count_nan_rows, \
+    drop_rows_containing_nans
+
+
+class TestNanWarnings(unittest.TestCase):
+    def test_raise_warning_if_nans_exist(self):
+        """
+        Test that a warning is raised if any nans are in the data
+        """
+        data = np.array([
+            [1, 2, 3],
+            [1, 2, np.nan],
+        ])
+
+        self.assertWarns(UserWarning, warn_if_nans_exist, data)
+
+    def test_count_rows_with_nans(self):
```

Comment:
Excellent cases - just wanted to suggest a few more; one where the # of rows is < 2 as per the condition in count_nan_rows. We should probably also send in a 1D vector as well to make sure that we can use this tool for y in addition to X?

Reply:
1. Fixed the row count (which was an error meant to check for dimensions).
2. Added a new function with a test that is for 1D arrays.

I'm going to modify the null remover to check across X and y correctly.